api-participant/src/main/java/bio/terra/pearl/api/participant/service/AuthUtilService.java

```diff
@@ -42,6 +42,16 @@ public Enrollee authParticipantUserToEnrollee(UUID participantUserId, String enr
     return enrolleeOpt.get();
   }
 
+  public PortalParticipantUser authParticipantUserToPortalParticipantUser(
+      UUID participantUserId, UUID portalParticipantUserId) {
+    return portalParticipantUserService
+        .find(portalParticipantUserId)
+        .filter(
+            portalParticipantUser ->
+                portalParticipantUser.getParticipantUserId().equals(participantUserId))
```

Comment:
BLOCKER: this needs to be broadened to cover proxies too.  Which is going to be a bit involved since proxies are at the enrollee level.  check out lines 100-106 of CurrentUserService for how to get all the proxy relationships for a portalParticipantUser.  Then you'll have to check whether the PortalParticipantUserId is associated with the ParticipantUserId of any of those proxied enrollees.